vue-app/src/utils/connection.ts

```diff
@@ -0,0 +1,26 @@
+import pkg from '@/../package.json'
+
+function lsGetKey(key: string) {
+  return `${pkg.name}.${key}`
+}
+
+export function lsSet(key: string, value: any) {
+  return localStorage.setItem(lsGetKey(key), JSON.stringify(value))
+}
+
+export function lsGet<T = any>(key: string, defaultValue: any = null): T {
+  const rawValue = localStorage.getItem(lsGetKey(key))
+  if (rawValue != null) {
+    try {
+      const value = JSON.parse(rawValue)
+      return value
+    } catch (e) {
+      return defaultValue
+    }
+  }
+  return defaultValue
+}
+
+export function lsRemove(key: string) {
+  return localStorage.removeItem(lsGetKey(key))
+}
```

Comment:
Everthing in here is related to local storage manipulation. I would call this file `localStorage.ts`.